script/openqa

```diff
@@ -15,6 +15,7 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
+use warnings;
```

Comment:
isn't that also implicit with using "Mojo::Base"?

Reply:
Yes, but apparently `Test::Strict` doesn't recognize it in this case.

Reply:
`use Mojo::Base -strict;` does indeed activate all strictures, including `warnings`.

Reply:
[Here's a list](https://metacpan.org/pod/Mojo::Base#DESCRIPTION) of the things the various `Mojo::Base` flags do. For a script like this `use strict; use warnings;` instead is totally fine. But i think all scripts should consistently use the same strictures activation.